monitoring/local_monitoring/common.sh

```diff
@@ -525,6 +525,69 @@ function kruize_local_demo_setup() {
 	fi
 }
 
+#setup the operator and deploy it
+operator_setup() {
+	echo "🔄 pulling the kruize operator repository"
+	git clone -b test-operator https://github.com/ncau/kruize-operator.git
+
+	echo "🔄 installing crds"
+	cd kruize-operator
+	make install kruize-operator
```

Comment:
Should this be just make install? Can you please run the demo and check if it works.